### PR TITLE
Cleaning up code

### PR DESCRIPTION
**Pull request by coder489**

### CODE REVIEW

**Comment by coder489** [comment]
image

**Comment by coder489** [comment]
Code Climate is integrated into this workflow, for some reason, it did not load on github, but in the details section, it shows that the code passed and there were no issues

File: shopping_cart_clean.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    time_now = time.strftime("%I:%M %p", t) 
+    return time_now
+
+def line():
+    """
+    Used to print the line for the receipt.
+    """
+    print("---------------------------------")
+
+def selected_products():
```
Comment (JLongarzo) [inline shopping_cart_clean.py:61]: Hi Analise, great work so far! I'm sure you just missed this but the selected_products() function is missing a parameter.

Comment (coder489) [reply shopping_cart_clean.py:61]: You are right! Thanks for the feedback!

**Pull request merged**